src/pageObject.js

```diff
@@ -1,16 +1,44 @@
 'use strict';
 
 class Header {
+  constructor(logo = 'Conduit logo') {
+    this.logo = logo;
+  }
 
-};
+  getLogo() {
+    return this.logo;
+  }
+}
 
 class PageObject {
+  constructor(url, header = new Header()) {
+    this.url = url;
+    this.header = header;
+  }
 
-};
+  clickOnLogo() {
+    return `Click on the ${this.header.getLogo()}`;
+  }
+}
 
 class ArticlePage extends PageObject {
+  constructor(url, slug, header = new Header()) {
+    super(url, header);
+    this.fullUrl = `${url}${slug}`;
+  }
 
-};
+  getCommentButtonLabel() {
+    return '[Publish comment] button';
```

Comment:
This method `getCommentButtonLabel` returns a static string. This could be converted to a static property of the class, or even a constant outside of the class, since it doesn't use any instance-specific data.